[2024-12-19T08:53:08.406+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:53:08.435+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2024-12-19T08:53:08.458+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2022-05-12T00:00:00+00:00 [queued]>
[2024-12-19T08:53:08.459+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T08:53:08.555+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2022-05-12 00:00:00+00:00
[2024-12-19T08:53:08.573+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4326) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:53:08.576+0000] {standard_task_runner.py:72} INFO - Started process 4338 to run task
[2024-12-19T08:53:08.589+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'scheduled__2022-05-12T00:00:00+00:00', '--job-id', '923', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp21pupfez']
[2024-12-19T08:53:08.594+0000] {standard_task_runner.py:105} INFO - Job 923: Subtask first_task
[2024-12-19T08:53:08.781+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task scheduled__2022-05-12T00:00:00+00:00 [running]> on host 8d142e9c83a7
[2024-12-19T08:53:09.165+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-05-12T00:00:00+00:00'
[2024-12-19T08:53:09.174+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:53:09.336+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T08:53:09.340+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T08:53:09.382+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T08:53:09.384+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T08:53:09.390+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T08:53:09.431+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:53:09.434+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=scheduled__2022-05-12T00:00:00+00:00, execution_date=20220512T000000, start_date=20241219T085308, end_date=20241219T085309
[2024-12-19T08:53:09.535+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T08:53:09.586+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:53:09.592+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
